src/Console/Command/CommandHelp.php

```diff
@@ -312,6 +319,39 @@ public static function getDisplayableAllowedValues(FixerOptionInterface $option)
         return $allowed;
     }
 
+    public static function getLatestReleaseVersionFromChangeLog()
+    {
+        static $version = null;
+
+        if (null !== $version) {
+            return $version;
+        }
+
+        $currentMajor = (int) Application::VERSION;
+        $changelogFile = __DIR__.'/../../../CHANGELOG.md';
+
+        if (!is_file($changelogFile)) {
+            return null;
+        }
+
+        $changelog = file_get_contents($changelogFile);
```

Comment:
Don't know if this may happen but I would either check if `false !== $changelog = @file_get_contents(...)` or add a `is_readable()` check before.